test/erlfmt_SUITE_data/rebar.config.script

```diff
@@ -0,0 +1,14 @@
+%% Example from https://github.com/benoitc/unicode_util_compat/blob/c39aed13801374aa577b5d75a93384276f49e8aa/rebar.config.script
```

Comment:
Oh wow, this probably has implications for insert_pragma, contains_pragma, but I can follow up in a different commit.

Reply:
I think it should be fine. There's still one node and we insert the comment before it. Since we support any node, not only an attribute as the first one, I think it should "just work".

Reply:
Ah yes, you made me fix that before.  Good point :)